test/signalrclienttests/connection_impl_tests.cpp

```diff
@@ -2017,3 +2017,65 @@ TEST(connection_id, connection_id_reset_when_starting_connection)
 
     ASSERT_EQ("", connection->get_connection_id());
 }
+
+TEST(connection_impl_stop, triggers_http_send_token)
```

Comment:
It might be interesting to add a test for calling stop on the signalr client while the initial WebSocket request is in flight.

Reply:
Figured that test existed and it does!
https://github.com/aspnet/SignalR-Client-Cpp/blob/0d51a369dca2b9c4dc539bb88a4476263e199409/test/signalrclienttests/connection_impl_tests.cpp#L1533